src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(...args) {
```

Comment:
The adder function is defined to accept a rest parameter, but according to the task description, it should only accept one argument at a time. You should remove the rest parameter syntax and define the function to accept a single argument.